include/Makefile.am

```diff
@@ -50,4 +50,5 @@ noinst_HEADERS = \
 	nccl-headers/neuron/error.h \
 	contrib/uthash.h \
 	contrib/utlist.h \
-	nccl_ofi_ep_addr_list.h
```

Comment:
In the "Move platform-specific hooks to nccl_ofi_platform.h" commit message, you need a body that explains why.

Reply:
I didn't have any better than having a clean separation for these hooks... but added to commit message.